### ISSUE TITLE
host port not configurable (and defaults to 3000)

### ISSUE THREAD

**Issue by redparchel** [hassio-addons/addon-happy-bubbles#2]
# Problem/Motivation

> When I install this I don't get the "Network" configuration option to change the Host port (and the port its using `3000` is conflicting with grafana in my case)

## Expected behavior

> The network configuration to be available and possibly the default to be 5555 (the happy bubbles default)

## Actual behavior

> Happy Bubbles uses port 3000 and is not configurable 

## Steps to reproduce

> Install the addon through the hassio addon store

## Proposed changes

> allow host port to be configurable, and possibly use the HB port default of 5555 (vs 3000)

it's also very possible I'm doing something wrong .. thanks for looking!

**Comment by frenck** [comment]
Hi there @redparchel,

I'm so sorry about this, I'm not sure how I did miss this.
I'm going to fix this as soon as possible and will take measures in my pre-flight checks to ensure this is not happing again in the future.

../Frenck

**Comment by redparchel** [comment]
Tossing my log in here:
```
Log
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-banner.sh: executing... 

-----------------------------------------------------------
 Hass.io Add-on: Happy Bubbles v0.2.0

 Presence detection server for Happy Bubbles Bluetooth beacon-based presence detection system.

 From: Community Hass.io Add-ons
 By: Franck Nijhof <frenck@addons.community>
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
Log level is set to INFO
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] 02-updates.sh: executing... 

INFO: You are running the latest version of this add-on
[cont-init.d] 02-updates.sh: exited 0.
[cont-init.d] 10-requirements.sh: executing... 
[cont-init.d] 10-requirements.sh: exited 0.
[cont-init.d] 11-nginx.sh: executing... 
[cont-init.d] 11-nginx.sh: exited 0.
[cont-init.d] 20-persist-db.sh: executing... 
[cont-init.d] 20-persist-db.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
 _   _    _    ____  ______   __  ____  _   _ ____  ____  _     _____ ____
| | | |  / \  |  _ \|  _ \ \ / / | __ )| | | | __ )| __ )| |   | ____/ ___|
| |_| | / _ \ | |_) | |_) \ V /  |  _ \| | | |  _ \|  _ \| |   |  _| \___ \
|  _  |/ ___ \|  __/|  __/ | |   | |_) | |_| | |_) | |_) | |___| |___ ___) |
|_| |_/_/   \_\_|   |_|    |_|   |____/ \___/|____/|____/|_____|_____|____/

 
CONNECTED TO MQTT

 
Visit http://0.0.0.0:3000 on your browser to see the web interface

 
2018/05/20 06:28:13 listen tcp 0.0.0.0:3000: bind: address already in use
```

Running hassio on a ubuntu 18.04 install (via the steps mentioned on the hassio wiki)

**Comment by frenck** [comment]
Thanks for providing the logs! @redparchel 👍

**Comment by redparchel** [comment]
thanks @frenck! really excited you've added Happy-Bubbles!

**Comment by frenck** [comment]
Fixed in `v0.3.0`.

Thanks for reporting @redparchel!
